elements/pfe-card/src/pfe-card.scss

```diff
@@ -50,32 +50,46 @@ $variables: (
   // Start of style declarations for host element
   display: flex;
   flex-direction: column;
-  justify-items: flex-start;
+  justify-items:    #{pfe-local($cssvar: JustifyItems, $fallback: flex-start)};
```

Comment:
I'm unsure how to test the justify properties, when I add any of these props to the card, nothing changes:
```
--pfe-card--JustifyItems: end;
--pfe-card--JustifyItems: flex-end;
--pfe-card--JustifyContent: space-around;
```
This one works great:
```
--pfe-card--AlignSelf: center;
```
however if i add the plain CSS to the card:
```
align-self: center;
``` 
it has the same effect. I'm wondering if we need CSS vars for properties on the host since they can be overridden with normal styles? They would be perfect for any styles on the card regions though.

Reply:
@castastrophe no rush if you haven't gotten to it yet, I just wanted to make sure you saw this and weren't waiting on me. Thx!